mxcube3/routes/Login.py

```diff
@@ -125,7 +125,6 @@ def loginInfo():
 
     login_info = login_info["loginRes"] if login_info is not None else {}
     login_info = limsutils.convert_to_dict(login_info)
-    limsutils.update_mxcube_session(login_id, login_info)
```

Comment:
Hm, well okey and what calls select_proposal ?, what hapends for instance when the server is restarted and we reconnect the client ? How is the mxcube Session object updated ?

Well in anyways, I guess Ill accept this and we have to fix it later ...